pkgs/misc/vim-plugins/overrides.nix

```diff
@@ -772,6 +772,10 @@ self: super: {
     ];
   });
 
+  vim-fzf-coauthorship = super.vim-fzf-coauthorship.overrideAttrs (old: {
+    dependencies = with self; [ vim-fzf ];
```

Comment:
```suggestion
    dependencies = with self; [ vim-fzf ];
```

Why is this so red?

Reply:
Probably non-ascii (white)space sneaked in? If that's what you're referring to. Good catch though, will fix!